src/viur/core/session.py

```diff
@@ -73,7 +72,11 @@ def load(self, req):
                     return False
 
                 self.cookie_key = cookie_key
-                self.session = data["data"]
+
+                # Here we have to use the pure db.Entity function to avoid changing the 'changed' flag
+                db.Entity.clear(self)
+                db.Entity.update(self, data["data"])
```

Comment:
I think a `super()` call would be more pythonic?
```suggestion
                super().clear()
                super().update(data["data"])
```

Reply:
applied and tested, worked.